src/main/resources/templates/humanreadable/measure_information_table.ftl

```diff
@@ -192,7 +192,13 @@
             <th scope="row" class="row-header"><span class="td_label">Definition</span></th>
             <td style="width:80%" colspan="3">
                 <div style="width:660px;">
-                    <pre>${model.measureInformation.definition!""}</pre>
+                    <#if model.measureInformation.definitions??>
+            			<#list model.measureInformation.definitions as definition>
+                    			${definition.term!""} - ${definition.definition!""} </br>
+            			</#list>
+        			<#else>
+                			${""}
+        			</#if>
```

Comment:
not sure if this is required

Reply:
@adongare Thanks Ashok! Do you think I should remove the else?

Reply:
if the list is empty anyway the div will be empty. so maybe not.

Reply:
@adongare ok thanks Ashok, so can I resolve the conversation and merge the code?

Reply:
Sorry, I meant yes. we can remove `else` part.